README.md

```diff
@@ -3,9 +3,9 @@
 Data Source is a JS library meant to help developers access Movable Ink Data Sources and the raw responses associated with their API integrations.
 
 ## Installation
-
+Add data sources to your package.json file. In devDependencies include the following:
 ```
-npm install data-source
+"data-source.js": "git@github.com:movableink/data-source.js.git"
```

Comment:
This should be this: 

`"data-source.js": "https://github.com/movableink/data-source.js.git#v0.0.3"`